modules/wolfgame.py

```diff
@@ -787,6 +795,189 @@ def fpinger(cli, nick, chan, rest):
     var.LAST_PING = None
     pinger(cli, nick, chan, rest)
 
+@cmd("pingif", "pingme", "pingat", pm=True)
+def altpinger(cli, nick, chan, rest):
+    """Pings you when the number of players reaches your preference."""
+    if not rest or (rest.isdigit() and int(rest) == 0):
+        if is_user_altpinged(nick)[0]:
+            msg = "Your ping preferences have been removed (was {0}).".format(is_user_altpinged(nick)[1])
+            if chan == nick:
+                pm(cli, nick, msg)
+            else:
+                cli.notice(nick, msg)
+            toggle_altpinged_status(nick, 0, is_user_altpinged(nick)[1])
+        elif not rest:
+            if chan == nick:
+                pm(cli, nick, "You need to enter a number.")
+            else:
+                cli.notice(nick, "You need to enter a number.")
+        else:
+            if chan == nick:
+                pm(cli, nick, "You do not have any preferences set.")
+            else:
+                cli.notice(nick, "You do not have any preferences set.")
+    elif rest.isdigit():
+        rest = int(rest)
+        if rest >= var.MAX_PLAYERS:
+            if chan == nick:
+                pm(cli, nick, "Please select a number between 0 and {0}.".format(var.MAX_PLAYERS-1))
+            else:
+                cli.notice(nick, "Please select a number between 0 and {0}.".format(var.MAX_PLAYERS-1))
+        elif is_user_altpinged(nick)[1] == rest:
+            if chan == nick:
+                pm(cli, nick, "Your ping preferences are already set to {0}.".format(rest))
+            else:
+                cli.notice(nick, "Your ping preferences are already set to {0}.".format(rest))
+        elif is_user_altpinged(nick)[0]:
+            msg = "Your ping preferences have been changed from {0} to {1}.".format(is_user_altpinged(nick)[1], rest)
+            if chan == nick:
+                pm(cli, nick, msg)
+            else:
+                cli.notice(nick, msg)
+            toggle_altpinged_status(nick, rest, is_user_altpinged(nick)[1])
+        else:
+            if chan == nick:
+                pm(cli, nick, "Your ping preferences have been set to {0}.".format(rest))
+            else:
+                cli.notice(nick, "Your ping preferences have been set to {0}.".format(rest))
+            toggle_altpinged_status(nick, rest)
+    else:
+        cli.notice(nick, "You need to enter a non-negative integer.")
+
+def is_user_altpinged(nick):
+    if nick in var.USERS.keys():
+        cloak = var.USERS[nick]["cloak"]
+        acc = var.USERS[nick]["account"]
+    else:
+        return False, None
```

Comment:
Style nit: If returning a tuple, please surround it in parenthesis. This makes it obvious to a casual observer that a) you are returning a tuple, and b) that returning a tuple was intentional as opposed to whatever else commas let you do